docker-compose.yml

```diff
@@ -8,22 +8,22 @@ services:
       dockerfile: Dockerfile
     image: ghcr.io/growlf/toolbox:latest
     environment:
-      - TESTVAR=${TESTVAR:-app}
+      - TESTVAR=${TESTVAR:/src}
     stdin_open: true
     tty: true
     command: ["/bin/zsh"]
     volumes:
-      - ${HOME}/.gitconfig:/home/ubuntu/.gitconfig
-      - ${HOME}/.ssh:/home/ubuntu/.ssh
-      - ${PWD}:/app
-      - /var/run/docker.sock:/var/run/docker.sock
+    - ${HOME}/.gitconfig:/home/ubuntu/.gitconfig
+    - ${HOME}/.ssh:/home/ubuntu/.ssh
+    - .:/src
```

Comment:
Generally better practice to use the shell variable and not globbing which might not act as expected.